win32/inc/azure_c_pal/lazy_init.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#ifndef LAZY_INIT_H
+#define LAZY_INIT_H
+
+#ifdef __cplusplus
+#include <cstdint>
+#else
+#include <stdint.h>
+#include <stdbool.h>
```

Comment:
>stdbool [](start = 10, length = 7)

doesn't seem to be used here

Reply:
removed.

---
In reply to: [466040910](https://github.com/Azure/azure-c-pal/pull/28#discussion_r466040910) [](ancestors = 466040910)